suites/pacific/rados/tier-3_rados_test-location-stretch-mode.yaml

```diff
@@ -151,20 +155,55 @@ tests:
   - test:
       name: Deploy stretch Cluster
       module: test_stretch_deployment_with_placement.py
-      polarion-id: CEPH-83573621
+      polarion-id: CEPH-83573621, CEPH-83572692
       config:
-        no_affinity: true
+        no_affinity: false
         stretch_rule_name: stretch_rule
         tiebreaker_mon_site_name: arbiter
       desc: Enables connectivity mode and deploys cluster with Stretch rule with arbiter node
       abort-on-fail: true
 
+  - test:
+      name: rbd-io
+      module: rbd_faster_exports.py
+      config:
+        rep-pool-only: True
+        rep_pool_config:
+          pool: rbd_rep_pool
+          image: rbd_rep_image
+          size: 10G
+        io-total: 100M
+      desc: Perform export during read/write,resizing,flattening,lock operations
+
+  - test:
+      name: rgw sanity tests
+      module: sanity_rgw.py
+      config:
+        script-name: test_multitenant_user_access.py
+        config-file-name: test_multitenant_access.yaml
+        timeout: 300
+      desc: Perform rgw tests
+
+  - test:
+      abort-on-fail: false
+      desc: "cephfs basic operations"
+      module: cephfs_basic_tests.py
+      name: cephfs-basics
+      polarion-id: "CEPH-11293"
+
+  - test:
+      name: nfs-ganesha_with_cephfs
+      module: nfs-ganesha_basics.py
+      desc: Configure nfs-ganesha on nfs server,do mount on any client and do IOs
+      polarion-id: CEPH-83574439
+      abort-on-fail: false
+
```

Comment:
can we have "abort-on-fail" set to "false" to all the above 4 tests? And remove the individual polarion ids as they are not the main scope of testing IMO. Makes sense?

Reply:
Will update in the next PR.